src/cascadia/TerminalControl/XamlUiaTextRange.cpp

```diff
@@ -4,7 +4,7 @@
 #include "pch.h"
 #include "XamlUiaTextRange.h"
 #include "UiaTextRange.hpp"
-
+#include <UIAutomationClient.h>

```

Comment:
interesting, why does this need .. this

Reply:
need this for `UIA_IsReadOnlyAttributeId` on line 93.

This was also a part of the FindText vs FindTextW mayhem